pipelines/datasets/au_abs_cpi/constants.py

```
@@ -0,0 +1,74 @@
+    )
+    LANDING_URL = (
+        "https://www.abs.gov.au/statistics/economy/price-indexes-and-inflation/"
+        "consumer-price-index-australia/latest-release"
+    )
+    # download.abs.gov.au / www.abs.gov.au reject the default requests UA.
+    HEADERS = {
+        "User-Agent": (
+            "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) "
+            "AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0 Safari/537.36"
+        )
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)